src/components/landing/layout/footer/language-picker.tsx

```diff
@@ -11,12 +11,21 @@ const LanguagePicker: React.FC = () => {
     await i18n.changeLanguage(event.currentTarget.value)
   }
 
+  const shortenLanguageCode = ():string => {
+    const language = i18n.language
+    if (language.substr(0, 2) === 'zh') {
+      return language
+    } else {
+      return language.substr(0, 2)
```

Comment:
why the extra handling of `'zh'`?

Reply:
Because there is zh-tw and zh-cn, but the handling is a bit odd :neutral_face:

Reply:
It's necessary. I need only the language from the 'language-area' code. Except for these two special cases.

Reply:
I would like another variant where these cases are determined automatically too, but this would probably be much more work and at least currently it's not really necessary.